include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -275,6 +275,9 @@
 #    if !(_ONEDPL_FPGA_DEVICE)
 #        define _USE_SUB_GROUPS 1
 #        define _USE_GROUP_ALGOS 1
+#        ifndef _ONEDPL_ICPX_USE_KNOWN_IDENTITY_FOR_ARITHMETIC_64BIT_DATA_TYPES
```

Comment:
I'm not sure the appropriate place to document the presence of such a PP define.  
How will a user know this feature is available?

I suppose we could add come CMake option, which then adds this to the cxx compiler flags.  On the other hand, perhaps this is too minor to include at that level, and users can just add their own flags.

Reply:
We will inform our users in documentation: which known issue we have and how to avoid it.